Ion.egg-info/PKG-INFO

```diff
@@ -1,47 +1,50 @@
-Metadata-Version: 1.1
+Metadata-Version: 2.1
 Name: Ion
 Version: 1.0
 Summary: The next-generation Intranet platform for TJHSST
 Home-page: https://github.com/tjcsl/ion
 Author: The TJHSST Computer Systems Lab
 Author-email: intranet@tjhsst.edu
 License: UNKNOWN
-Description: **********
-        Intranet 3
-        **********
-        .. image:: https://github.com/tjcsl/ion/workflows/CI/badge.svg?branch=master
-            :target: https://github.com/tjcsl/ion/actions
-            :alt: CI
-        
-        .. image:: https://coveralls.io/repos/tjcsl/ion/badge.svg?branch=master&service=github
-            :target: https://coveralls.io/github/tjcsl/ion?branch=master
-            :alt: Coverage
-        
-        *Version 3.0.0*
-        
-        Intranet3 (Ion) is the next-generation Intranet platform for `TJHSST 
-        <https://www.tjhsst.edu/>`_. Using Python, Django, Redis, Postgres, and many other technologies, Ion was developed from the ground up to be simple, well-documented, and extensible.
-        
-        Documentation (in RestructuredText format) is available inside the "docs" folder or at https://tjcsl.github.io/ion publicly on the web.
-        
-        **What does the TJ Intranet do?** Ion allows students, teachers, and staff at TJHSST to access student information, manage activity signups for TJ's Eighth Period program, and view information on news and events. `Read more about how Ion is used at Thomas Jefferson <https://ion.tjhsst.edu/about>`_.
-        
-        **Ion now requires Python 3.7+** Python 3.8.5 is currently used in both production and testing environments.
-        
-        **How can I create a testing environment?** Read the section on `Setting up Vagrant <https://tjcsl.github.io/ion/setup/vagrant.html>`_ in the documentation.
-        
-        **How can I contribute to Ion? How can I report a bug? How can I report a security issue/vulnerability?**
-        
-        Please read our `contributing guide <https://github.com/tjcsl/ion/blob/master/CONTRIBUTING.md>`_ for more information.
-        
-        Current Intranet maintainer: `Laur04 <https://github.com/Laur04>`_
-        
-        Intranet maintainer emiriti: `Theo Ouzhinski <https://github.com/theo-o>`_ (TJ 2020) & `anonymoose2 <https://github.com/anonymoose2>`_
-        
 Platform: UNKNOWN
 Classifier: Development Status :: 5 - Production/Stable
 Classifier: License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)
 Classifier: Operating System :: POSIX :: Linux
 Classifier: Programming Language :: Python :: 3.6
 Classifier: Programming Language :: Python :: 3.7
 Classifier: Framework :: Django :: 1.11
+License-File: COPYING
```

Comment:
Any particular reason for moving this block? Just curious; I'm not super familiar with the convention here, but this part LGTM.

Reply:
It's apparently that way the 2.1 metadata format works, which setuptools 57+ picks by default.